main.js

```diff
@@ -1 +1,50 @@
 import customers from './customers.js'
+
+
+// let newCustomer = document.querySelector('#name-list')
+
+// console.log()
+
+// let newPerson = document.createElement("li")
+// let personName = document.createTextNode("Mr. Curtis Ryan")
+
+// let personNumber = document.createTextNode(" ")
```

Comment:
You want to start with `document.createElement('p')` or whatever element you want, *then* create a text node and append it if the element needs text in it.